nuxt-pinia-tailwind/components/UserRepos/UserRepos.vue

```diff
@@ -1,13 +1,46 @@
 <template>
   <div class="w-full border rounded-lg relative bg-white">
+    <div class="flex items-center justify-between px-4 py-3">
+      <h3 class="font-semibold">Repositories</h3>
+    </div>
+
     <RepositoryCard
-      name="name-of-the-repo"
-      :is-private="false"
-      description="this is a description"
+      v-for="repo in repos"
+      :key="repo.id"
+      :name="repo.name"
+      :is-private="repo.private"
+      :description="repo.description"
     />
+
+    <div class="flex items-center justify-center py-4">
+      <NuxtLink :to="'/' + username" class="text-blue-500 font-semibold"
+        >View all repositories</NuxtLink
+      >
+    </div>
   </div>
 </template>
 
 <script lang="ts">
-export default {};
+import { useAsync, useContext } from '@nuxtjs/composition-api';
+import Vue from 'vue';
+import RepositoryCard from '../RepositoryCard/RepositoryCard.vue';
```

Comment:
Lets do an alias here, instead of `../` lets  do: 

```suggestion
import RepositoryCard from '~/components/RepositoryCard/RepositoryCard.vue;
```